server/events.lua

```diff
@@ -7,8 +7,10 @@ AddEventHandler('playerDropped', function(reason)
 
     local wasReleased, userRef = ReleasePlayerUserAsDisconnected(source, reason)
 
-    local Character = User:Character()
-    Character:SetLastPosition( playerPosition )
+    if User then 
```

Comment:
Need move this code below to inside this method `ReleasePlayerUserAsDisconnected`.
Becouse the user is being deleted within this method

Move this code below too
```
User:Save()
User:ClearCache()
```

Reply:
I updated it. I hope I understood correctly.